src/js/dropdown.js

```diff
@@ -259,6 +275,8 @@ var DropDown = (function(DX) {
 			if (!isShownOnce) {
 				isShownOnce = true;
 				reCalculateWidth(block, control, config);
+				calculateAnimationDelay(block);

```

Comment:
Rename calculateAnimationDelay, calculateHeight to reCalculateAnimationDelay, reCalculateHeight  
to be in uniform style with old code